### DISCUSSION TITLE
How to restore

### DISCUSSION THREAD

**Discussion by gerardo-cipriano-dinova** [cooperspencer/gickup#244] [Q&A] [answered]
Hi,
How do I restore my repo from this archive?
![image](https://github.com/cooperspencer/gickup/assets/151726214/52b3ba2b-aa30-4ada-ae34-cfc265a796a8)

**Comment by cooperspencer** [comment, marked as answer]
First of all you need to unpack this repository. After that you'll change into the unpacked repository and do a `git push --all --force` and a `git push --tags --force` and this will push the latest backup to the repository.
But be careful with that, because it will overwrite your current repo.

**Comment by rohanod** [comment]
Hi, I know this is probably a stupid question, but how do I unpack the repo?

**Reply by cooperspencer** [reply]
@rohanod how did you do the backup?
